src/test/java/net/sf/jabref/model/entry/BibEntryTests.java

```diff
@@ -212,16 +212,21 @@ public void setNullField() {
 
     @Test(expected = NullPointerException.class)
     public void addNullKeywordThrowsNPE() {
-        keywordEntry.addKeyword(null);
+        keywordEntry.addKeyword(null, Globals.prefs.get(JabRefPreferences.KEYWORD_SEPARATOR));
```

Comment:
Well, there are many tests that rely on default preferences without
specifying it. For example, the begin and end symbols for field `{` and `}`
are from the preferences and using an empty `SavePreferences` will
indirectly cause a requirement of specifying these (and, hence, it would
have been better to initialize those from the preferences in all tests).
I'm not really sure about all the cases here but somehow it may be better
to avoid later surprises...

Either way: not very hard to search and replace later. I'm travelling so
most likely nothing will be changed until tomorrow evening at the latest.